meetup/services/generic_api.py

```diff
@@ -0,0 +1,21 @@
+from django.http import HttpResponse
+import requests
+from meetuppizza.settings import base
+
+class GenericApi():
+  BASE_URL = ''
+
+  def __init__(self, link):
```

Comment:
That this takes a link on initialization makes me wonder if this class is misnamed. Is it really a `GenericApi`, or is it something more specific? It seems to me that its responsibilities revolve more around fetching API details for one particular entity.